services/indexes/pvm/writer.go

```diff
@@ -78,83 +81,93 @@ func NewWriter(networkID uint32, chainID string) (*Writer, error) {
 		chainID:     chainID,
 		networkID:   networkID,
 		avaxAssetID: avaxAssetID,
-		codec:       platformvm.Codec,
+		codec:       txs.Codec,
 		avax:        avaxIndexer.NewWriter(chainID, avaxAssetID),
 		ctx:         ctx,
 	}, nil
 }
 
 func (*Writer) Name() string { return "pvm-index" }
 
-func (w *Writer) initCtxPtx(p *platformvm.Tx) {
-	switch castTx := p.UnsignedTx.(type) {
-	case *platformvm.UnsignedAddValidatorTx:
-		for _, utxo := range castTx.UTXOs() {
+func (w *Writer) initCtxPtx(p *txs.Tx) {
+	switch castTx := p.Unsigned.(type) {
+	case *txs.AddValidatorTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
-		for _, utxo := range castTx.Stake {
+		for _, utxo := range castTx.StakeOuts {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedAddSubnetValidatorTx:
-		for _, utxo := range castTx.UTXOs() {
+	case *txs.AddSubnetValidatorTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedAddDelegatorTx:
-		for _, utxo := range castTx.UTXOs() {
+	case *txs.AddDelegatorTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
-		for _, utxo := range castTx.Stake {
+		for _, utxo := range castTx.StakeOuts {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedCreateSubnetTx:
-		for _, utxo := range castTx.UTXOs() {
+	case *txs.CreateSubnetTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedCreateChainTx:
-		for _, utxo := range castTx.UTXOs() {
+	case *txs.CreateChainTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedImportTx:
-		for _, utxo := range castTx.UTXOs() {
+	case *txs.ImportTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		for _, out := range castTx.Outs {
 			out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedExportTx:
-		for _, utxo := range castTx.UTXOs() {
+	case *txs.ExportTx:
+		for _, utxo := range castTx.Outs {
 			utxo.Out.InitCtx(w.ctx)
 		}
 		for _, out := range castTx.ExportedOutputs {
 			out.InitCtx(w.ctx)
 		}
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedAdvanceTimeTx:
+	case *txs.AdvanceTimeTx:
 		castTx.InitCtx(w.ctx)
-	case *platformvm.UnsignedRewardValidatorTx:
+	case *txs.RewardValidatorTx:
 		castTx.InitCtx(w.ctx)
 	default:
 	}
 }
 
-func (w *Writer) initCtx(b platformvm.Block) {
+func (w *Writer) initCtx(b blocks.Block) {
```

Comment:
This function shouldn't exist: https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/blocks/block.go#L17

Reply:
Same question.  I'm not really sure what snow.ContextInitializable.  I'm guessing from context that it pre-inits